### PR TITLE
[NCL-7224] Implement Product Milestone detail page

### PR DESCRIPTION
**Pull request by patrikk0123**

### CODE REVIEW

File: src/components/ProductMilestoneDetailPage/ProductMilestoneDetailPage.tsx [hunk 1]
```diff
@@ -1,9 +1,68 @@
...
+
+  const attributes = [
+    {
+      name: 'Status',
+      value: serviceContainerMilestone.data?.endDate ? 'CLOSED' : 'OPEN',
+    },
+    {
+      name: 'Start Date',
+      value: (
+        <OptionalText>{createDateTime({ date: serviceContainerMilestone.data.startingDate, includeTime: false })}</OptionalText>
```
Comment (DnsZhou) [inline src/components/ProductMilestoneDetailPage/ProductMilestoneDetailPage.tsx:40]: The date format is different from our angularjs version, are we going to refactor them in a new way or we should keep it as the same one with before?
image
image

@matedo1

Comment (patrikk0123) [reply src/components/ProductMilestoneDetailPage/ProductMilestoneDetailPage.tsx:40]: This format is used in **createDateTime** function so I think it is correct.

**Review by DnsZhou — APPROVED** [review]
All the code part looks good to me except the date format.

**Review by matedo1 — CHANGES_REQUESTED** [review]
Just small changes, thanks

**Review by matedo1 — APPROVED** [review]

**Pull request merged**